### ISSUE TITLE
Co-WIN Public API's cached response

### ISSUE THREAD

**Issue by meetmakwana19** [cowinapi/developer.cowin#541]
My python script runs for 24hours at an interval of 3 seconds due to 100 requests im 5minutes criteria. The fetched response I'm getting inside my DB is usually 3-5-10 minutes late than the response which telegram-vaccine-slots-notifier(Eg.: Healthify one) gets.

I am attaching images here which shows that on 19th September 2021, 
Healthiy telegram tracker notified the slot at **_8:20am_**  with **_Dose1:40, Dose2:40_**
whereas my database got the insertion of the response at **_8:21am_** with **_Dose1:24, Dose2:21_**
Conclusion : I am getting cached response from server side side cachingI think.

I tried adding headers for no-caching in the python code but isn't working too. If anyone has any suggestions on how to avoid cached response from cowin public api then please help.

![4e131e76-79c4-44c1-b132-a54625271d2e](https://user-images.githubusercontent.com/72190681/134082615-8e020df4-4e39-46ae-aa5c-f656d068e958.jpg)
![ffff83ee-d4d4-40d2-bbcd-db5687673b0c](https://user-images.githubusercontent.com/72190681/134082658-a21e75ac-7ad1-48ec-99da-266460680703.jpg)

**Comment by siddhesh1770** [comment]
Yes, this is known issue and no one cares about fixing the same.

**Comment by meetmakwana19** [comment]
> Yes, this is known issue and no one cares about fixing the same.

It would really be great if you know any of the solution. I use calendarbyDistrict api

**Comment by siddhesh1770** [comment]
You can use api endpoints of healthifyme unofficially like this, they have private access to Cowin data.

https://www.healthifyme.com/cowin/booking/meta/dayWiseSlots?dose=dose_1&language=en&district_id=393

**Comment by meetmakwana19** [comment]
> You can use api endpoints of healthifyme unofficially like this, they have private access to Cowin data.
> 
> https://www.healthifyme.com/cowin/booking/meta/dayWiseSlots?dose=dose_1&language=en&district_id=393

Can you please share the documentation of this Healthify's APIs as I couldn't find it. I want to know other endpoints like you had given example of dayWiseSlots. Your provided URL just returns a total present number of appointments on the CoWIN but I need exact data like CoWIN gives from their other endpoints like findbydistrict or calenderbydistrict.

**Comment by siddhesh1770** [comment]
as i said this is unofficial way so there is no documentation, Just go to [https://apps.healthifyme.com/vaccinateme](url) and use browser dev options to track different endpoints and its params and headers
